productPage/productPage.css

```diff
@@ -0,0 +1,242 @@
+@import url('https://fonts.googleapis.com/css2?family=Karla:wght@200;300;400;500;600;700;800&family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+/* * {
+    font-family: 'Montserrat', 'sans-serif';
+} */
+
+.grid-container {
+    display: grid;
+    grid-template-columns: (3, 1fr);
+    grid-template-areas:
+        'navbar navbar navbar'
+        'sidebar content content'
+        'sidebar content content'
+        'sidebar content content'
+}
+
+.header {
+    grid-area: navbar;
+    /* border: 2px solid red; */
```

Comment:
You must remove these comments if not needed now. Unwanted comments make your code looks messy

Reply:
done